Users/Tutorial.rst

```diff
@@ -292,10 +303,26 @@ After the results we've already seen, we'll see a new informational one
 which informs us that we have a TODO in our code.
 
 Did you note that we didn't specify which files to check this time? This
-is because all settings, including ``files = src/*.c``, from the Default
-section are already available in every other section implicitly. Thus
-the default section is a good point to set things like logging and
-output settings or specifying a default set of files to check.
+is because all settings, including ``files = src/*.c``, from the ``all``
+section (previously called ``cli``) have been inherited in the new
+``TODOS`` section that we just added.
+
+You can make a section inherit from any previously defined section using
+this syntax:
+
+::
+
+    [parentSection.childSection]
+
+.. note::
+
+    ``cli`` is an internally reserved section name. All of its settings
+    are implicitly inherited to every other section by default. It is
+    because of this implicit inheritance feature that we are adviced to
+    rename the ``cli`` section to something else. Doing so will save us
+    from having unexpected values of ``cli`` being implicitly inherited
+    into our sections. We suggest that you change its name to ``all``.
```

Comment:
Hm not sure we should encourage people to name it "all". But yeah, maybe a good convention in future? Not sure :3